gpkit/constraints/sgp.py

```diff
@@ -108,6 +105,8 @@ def localsolve(self, solver=None, verbosity=1, x0=None, reltol=1e-4,
             print("Beginning signomial solve.")
         self.gps = []  # NOTE: SIDE EFFECTS
         self.results = []
+        if x0 and mutategp:
+            self._gp = self.init_gp(self.substitutions, verbosity, x0)
```

Comment:
I still don't understand. Let's say I have an SP called `sp`, and an initial guess `initial_guess`. Can we agree that 

`sp.localsolve(x0=initial_guess, mutategp=False)`

will return exactly the same thing as

`sp.localsolve(x0=None, mutategp=False)`
```
?

That's certainly what it looks like if I follow the code, and I'd be sad as a user to have my specified `x0` ignored.